packages/itwin/reports-config-widget/src/widget/context/ApiContext.tsx

```diff
@@ -0,0 +1,27 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+import type { AccessToken } from "@itwin/core-bentley";
+import * as React from "react";
+import { createContext } from "react";
+
+export interface ApiConfig {
```

Comment:
name still too generic imo, that error you throw would confuse me, how does ReportsApiConfig sound?